libraries/standard/mqtt/include/mqtt.h

```diff
@@ -160,9 +160,14 @@ MQTTStatus_t MQTT_Init( MQTTContext_t * pContext,
  * @brief param[in] pContext Initialized MQTT context.
  * @brief param[in] pConnectInfo MQTT CONNECT packet parameters.
  * @brief param[in] pWillInfo Last Will and Testament. Pass NULL if not used.
- * @brief param[in] timeoutMs Timeout in milliseconds for receiving
- * CONNACK packet. A zero timeout value will attempt a single reception
- * from the transport interface.
+ * @brief param[in] timeoutMs Timeout in milliseconds for receiving CONNACK
+ * packet. In certain conditions, depending on the network conditions or the
+ * transport interface receive timeout configured, this function can spend more
+ * time in receiving CONNACK packet as compared to the timeout provided. However
```

Comment:
(Nit): "..more time in receiving CONNACK packet ~~as compared to the~~ than the provided timeout duration. However, (comma)...."